packages/lib/sdk/src/usql/query/Query.spec.js

```diff
@@ -589,16 +589,16 @@ describe('Query', () => {
 			expect(q.lengthLoaded).toBe(true);
 			expect(q.length).toBe(0);
 		});
-		it('should not execute the query, and ignore initial data if noResolve is set', () => {
+		it('should not execute the query and use initial data, even if noResolve is set', () => {
```

Comment:
This test name needs some revision, and the test itself could be improved too

**`should not execute the query`**
We need an assertion for this. Ideally expecting some function to not be called, or expecting some state to indicate the query hasn't been executed.

**`should not ... and use initial data`**
Can you clarify, does this mean we should use initial data, or should not? Either way, we need an assertion for this.

**`even if noResolve is set`**
This phrasing makes me think that `noResolve` isn't important here, and the outcome of the test will be the same *`even`* if `noResolve` is `false`. Is this the case? If so, we don't need to manually set `noResolve` in the test. If not, this test name should use *`if`* not *`even`*, like it was before.